docs/guides/integration-prebid.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Prebid Integration
+description: Information about integrating with Prebid as part of your UID2 implementation.
+hide_table_of_contents: false
+sidebar_position: 04
+---
+
+# Prebid Integration Guide
+
+<!-- This guide includes the following information:
+
+- [Prebid Overview](#prebid-overview)
+- [UID2 Page on the Prebid Site](#uid2-page-on-the-prebid-site)
+- [UID2 User ID Submodule](#uid2-user-id-submodule)
+- [Client Refresh Mode](#client-refresh-mode)
+  -  [Client Refresh Cookie Example](#client-refresh-cookie-example)
+  -  [Client Refresh uid2Token Example](#client-refresh-uid2token-example)
+- [Server-Only Mode](#server-only-mode)
+   - [Server-Only Cookie Example](#server-only-cookie-example)
+   - [Server-Only Value Example](#server-only-value-example)
+- [Storage of Internal Values](#storage-of-internal-values)
+- [Sample Token](#sample-token)
+- [Prebid Implementation Notes and Tips](#prebid-implementation-notes-and-tips) -->
+
+## Prebid Overview
+
+Prebid is a premier, very popular, free, open source header bidding solution.
+
+If you are a publisher who is using header bidding using Prebid, or wants to do so, there are a few extra steps so that your Prebid header bidding implementation also supports UID2.
+
+In addition, if you don't already have one, you must set up a UID2 account: see [Account Setup](../getting-started/gs-account-setup.md).
+
+## UID2 Page on the Prebid Site
+
+For details about the UID2 Prebid integration, refer to [the Unified ID 2.0 page on the Prebid site](https://docs.prebid.org/dev-docs/modules/userid-submodules/unified2.html). 
+Be sure to follow all the steps.
+
+## UID2 User ID Submodule
+
+UID2 requires initial tokens to be generated server-side. The UID2 module handles storing, providing, and optionally refreshing them. The module can operate in one of two different modes:
+
+- Client Refresh
+- Server-Only
+
+  *Server-Only* mode was originally called *legacy* mode, but it is a popular mode for new integrations where publishers prefer to handle token refresh server-side.
```

Comment:
let's remove _Legacy Mode_/Server-only mode here and below as Dana suggested for now to simplify? If you just want to merge this quickly then leave it unchanged it's fine.

Reply:
@sunnywu I removed it. It's a chunk of info including code examples. The doc is very short now!

Reply:
Done. Removed a LOT of stuff that went with that.